common/src/main/resources/quartz.properties

```diff
@@ -2,6 +2,8 @@ spring.quartz.job-store-type=jdbc
 spring.quartz.overwrite-existing-jobs=true
 spring.quartz.properties.org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
 spring.quartz.properties.org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreTX
+spring.quartz.properties.org.quartz.scheduler.instanceName=cbQuartzStore
```

Comment:
Why do we need to add this? I mean doesn't it break the scheduling of old clusters?